src/configParamsJSON/imageryViewerConfig.json

```diff
@@ -563,31 +563,38 @@
       "content": [
         {
           "type": "subsection",
-          "id": "themeLayout",
+          "id": "theme",
           "content": [
             {
               "type": "setting",
-              "id": "applySharedTheme",
+              "id": "theme",
               "express": true,
-              "defaultValue": true
+              "defaultValue": "light"
             },
             {
               "type": "setting",
-              "id": "headerColor",
+              "id": "customTheme",
               "express": true,
-              "defaultValue": "#ffffff"
-            },
-            {
-              "type": "setting",
-              "id": "headerBackground",
-              "express": true,
-              "defaultValue": "#0079c1"
+              "defaultValue": null,
+              "config": {
+                "singleFont": false
+              }
             },
             {
               "type": "setting",
               "id": "customCSS",
-              "express": false,
               "defaultValue": ""
+            },
+            {
```

Comment:
@adithya-gopakumar  I only see mapToolsExpanded in Attachment Viewer settings. is this a property you meant to add to imagery viewer?

Reply:
No. That was a mistake. I have removed it now.